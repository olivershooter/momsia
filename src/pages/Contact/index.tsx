import React from 'react';
import { Typography } from '@mui/material';

const Contact = () => {
  return <Typography sx={{ color: 'black' }}>contact</Typography>;
};

export default Contact;
