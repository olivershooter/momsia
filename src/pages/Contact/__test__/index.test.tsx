import { BrowserRouter } from 'react-router-dom';
import { describe, expect, it } from 'vitest';
import Contact from '..';
import React from 'react';
import { render } from '@testing-library/react';

describe('Contact page', () => {
  it('renders and matches the snapshot', () => {
    const { asFragment } = render(
      <BrowserRouter>
        <Contact />
      </BrowserRouter>,
    );

    expect(asFragment()).toMatchSnapshot();
  });
});
