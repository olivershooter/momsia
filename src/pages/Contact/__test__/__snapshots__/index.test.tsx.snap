// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Contact page > renders and matches the snapshot 1`] = `
<DocumentFragment>
  <div>
    contact me pls
  </div>
</DocumentFragment>
`;
